### PR TITLE
fix : remove provider reloads

### PR DESCRIPTION
**Pull request by jabahum**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

### CODE REVIEW

File: packages/esm-patient-queues-app/src/active-visits/change-status-dialog.component.tsx [hunk 1]
```diff
@@ -66,20 +63,35 @@ const ChangeStatus: React.FC<ChangeStatusDialogProps> = ({ queueEntry, currentEn
...
-        critical: true,
-        description: error?.message,
-      });
-    },
-  );
+  const [isLoading, setIsLoading] = useState(true);
+
+  const providerUuid = useMemo(() => {
+    if (!sessionUser?.user?.uuid) return null;
+    setIsLoading(true);
```
Comment (Daphne210) [inline packages/esm-patient-queues-app/src/active-visits/change-status-dialog.component.tsx:70]: By default, you have set the state of isLoading to true and you're setting it here again as true, was this intentional or it should be the other way round?

Comment (jabahum) [reply packages/esm-patient-queues-app/src/active-visits/change-status-dialog.component.tsx:70]: intentional

File: packages/esm-patient-queues-app/src/active-visits/change-status-move-to-next-dialog.component.tsx [hunk 2]
```diff
@@ -39,9 +40,12 @@ const ChangeStatusMoveToNext: React.FC<ChangeStatusDialogProps> = ({ patientUuid
   const { t } = useTranslation();
 
   const sessionUser = useSession();
+  let isCancelled = false;
```
Comment (Daphne210) [inline packages/esm-patient-queues-app/src/active-visits/change-status-move-to-next-dialog.component.tsx:43]: Added this but used it no where

File: packages/esm-patient-queues-app/src/active-visits/pick-patient-dialog.component.tsx [hunk 3]
```diff
@@ -39,39 +17,45 @@ interface PickPatientDialogProps {
 const PickPatientStatus: React.FC<PickPatientDialogProps> = ({ queueEntry, closeModal }) => {
   const { t } = useTranslation();
 
-  const locations = useLocations();
-
-  const [selectedLocation, setSelectedLocation] = useState('');
+  let isCancelled = false;
```
Comment (Daphne210) [inline packages/esm-patient-queues-app/src/active-visits/pick-patient-dialog.component.tsx:20]: Likewise here

File: packages/esm-patient-queues-app/src/active-visits/change-status-move-to-next-dialog.component.tsx (hunk 2 above)
Comment (jabahum) [reply packages/esm-patient-queues-app/src/active-visits/change-status-move-to-next-dialog.component.tsx:43]: needs to be removed

File: packages/esm-patient-queues-app/src/active-visits/pick-patient-dialog.component.tsx (hunk 3 above)
Comment (jabahum) [reply packages/esm-patient-queues-app/src/active-visits/pick-patient-dialog.component.tsx:20]: same here

File: packages/esm-patient-queues-app/src/active-visits/change-status-dialog.component.tsx (hunk 1 above)
Comment (Daphne210) [reply packages/esm-patient-queues-app/src/active-visits/change-status-dialog.component.tsx:70]: Okay then. Do you mind pulling the latest changes from master am getting 179 conflicts in the yarn.lock file

File: packages/esm-patient-queues-app/src/active-visits/change-status-move-to-next-dialog.component.tsx (hunk 2 above)
Comment (Daphne210) [reply packages/esm-patient-queues-app/src/active-visits/change-status-move-to-next-dialog.component.tsx:43]: Okay will leave it you to you

File: packages/esm-patient-queues-app/src/active-visits/change-status-dialog.component.tsx (hunk 1 above)
Comment (jabahum) [reply packages/esm-patient-queues-app/src/active-visits/change-status-dialog.component.tsx:70]: you definitely have a separate lock file.

Comment (Daphne210) [reply packages/esm-patient-queues-app/src/active-visits/change-status-dialog.component.tsx:70]: Because my lock file is up to date.

**Review by akileng56 — APPROVED** [review]

**Pull request merged**